analysisOnData/runAnalysisOnWJetsMC.py

```diff
@@ -48,7 +48,7 @@ def RDFprocessWJetsMC(fvec, outputDir, sample, xsec, fileSF, ncores, pretendJob=
             p.branch(nodeToStart = 'defs', nodeToEnd = '{}/prefit_{}/Nominal'.format('WToTau', region), modules = [ROOT.muonHistos(wtotau_cut, weight, nom,"Nom",0)])     
             
             #reco templates with AC reweighting
-            steps = [ROOT.getACValues(fileAC),ROOT.getMassWeights(),ROOT.getWeights(),ROOT.templateBuilder()]
+            steps = [ROOT.getACValues(fileAC),ROOT.defineHarmonics(),ROOT.getMassWeights(),ROOT.getWeights(),ROOT.templateBuilder(wtomu_cut, weight,nom,"Nom",3)]
```

Comment:
@sroychow here histocategory is 0! 3 is for weight variations below (with the relevant weight systematics)